### PR TITLE
New

### PR DESCRIPTION
**Pull request by prats7up**

### CODE REVIEW

File: test-file [hunk 1]
```diff
@@ -1 +1,3 @@
 This is some dummy text!
+
+INDEED
```
Comment (dutta-alankar) [inline test-file:3]: As you can see that there are no conflicts in here and this PR can be merged!

File: TEST/test.tex [hunk 2]
```diff
@@ -0,0 +1 @@
+Thanks Alankar!
```
Comment (dutta-alankar) [inline TEST/test.tex:1]:
```suggestion
Thanks Alankar!
This is how I can suggest edits n the code and you can commit this to your forked branch before this PR gets merged.
```

**Comment by dutta-alankar** [comment]
These are my initial review comments!

**Review by prats7up — COMMENTED** [review]
I have reviewed all your suggestion. Pls merge.

**Review by dutta-alankar — APPROVED** [review]

**Pull request merged**